stablehlo/tests/interpret_abs.mlir

```diff
@@ -9,6 +9,24 @@ func.func @abs_op_test_si64() {
 
 // -----
 
+func.func @abs_op_test_f8_e4m3_fnuz() {
+  %operand = stablehlo.constant dense<[24.0, -24.0, -0.0, 0x80]> : tensor<4xf8E4M3FNUZ>
+  %result = stablehlo.abs %operand : tensor<4xf8E4M3FNUZ>
+  check.expect_almost_eq_const %result, dense<[24.0, 24.0, 0.0, 0x80]> : tensor<4xf8E4M3FNUZ>
+  func.return
+}
```

Comment:
@sdasgup3 What's our policy for updating interpreter tests for new types? If I recall correctly from the [Q&A](https://github.com/openxla/stablehlo/blob/main/docs/reference.md#testing-guidelines), we only add test for highest bitwidth types, and rely on APInt/APFloat to handle the general case?

Reply:
@jakeh-gc Thanks for the interpreter support and tests.  

As the newly introduced types are handled uniformly just like other floating point types in the context of add op, I would recommend to stick to the guideline [G1](https://github.com/openxla/stablehlo/blob/main/docs/reference.md#testing-guidelines), that Eugene pointed out. With that we can skip adding extra tests here and for other ops.

>> I'm happy to drop them.
Thanks!

>>  There are code paths only exercised by these types.

You are absolutely right in pointing that! 
For every op we have a code path, in Tensor::set and Tensor::get (exactly as you pointed), which basically packs and unpacks the tensor data and by doing so  "covers" all the types. As the interpreter tests are written based on meeting code coverage, this means for every op we should write tests for every type, which is not valuable and hard to maintain. 

>> Clearly, 2 could be covered by just a few cases.

That is exactly the path we followed!
 
To avoid the problem, mentioned above, we decided, via [G3](https://github.com/openxla/stablehlo/blob/main/docs/reference.md#testing-guidelines) to write dedicated tests  just for this packing/unpacking code and thereby reduce the burden on op level tests.

Reply:
With the exception of `constant`, I've removed the interpreter test cases.